ecommerce/serializers.py

```diff
@@ -22,6 +22,27 @@
 from mitxpro.serializers import WriteableSerializerMethodField
 
 
+def _get_thumbnail_url(page):
```

Comment:
Don't duplicate this method

Reply:
Removed, was left over from some fixes earlier, isn't being used anyway. Oversight on my part.